app/models/storage.rb

```diff
@@ -70,6 +71,12 @@ class Storage < ApplicationRecord
 
   SUPPORTED_STORAGE_TYPES = %w( VMFS NFS FCP ISCSI GLUSTERFS )
 
+  supports :smartstate_analysis do
+    if ext_management_systems.blank? || !ext_management_system.kind_of?(ManageIQ::Providers::Vmware::InfraManager)
```

Comment:
`!ext_management_system.kind_of?(ManageIQ::Providers::Vmware::InfraManager)`  this should be removed.

For some reason this code expects only hosts that are connected to Vmware Inframanger to support SSA. 
So this logic should go into Vmware Inframanager. 

@agrare do you know what a provider needs to implement to support ssa for a datastore? Or do you know who could know? :smile: